.luacheckrc

```diff
@@ -26,4 +26,6 @@ include_files = {
 
 exclude_files = {
     "lib/tarantool-python",
+    "test/test-tarantool/panic_on_broken_lsn.test.lua",
+    "test/test-tarantool/setopt_delimeter.test.lua",
```

Comment:
I think we should exclude the whole test suite, because it is 'core = tarantool' (so, it is not a lua interpreter input, but tarantool console input).

Reply:
Excluded by mask.